src/kiota/Handlers/Plugin/AddHandler.cs

```diff
@@ -23,6 +24,16 @@ public required Option<List<PluginType>> PluginTypesOption
     {
         get; init;
     }
+
+    public required Option<SecuritySchemeType> PluginAuthTypeOption
```

Comment:
I think this should be a plugin manifest auth type and not an OpenAPI security scheme type. @maisarissi, what's your opinion

Reply:
According to this [comment](https://github.com/microsoft/kiota/issues/5072#issuecomment-2265907850) on the issue discussion, what is expected is a supported security scheme type

Reply:
Yes, the input is defined in https://github.com/microsoft/kiota/issues/5070 (linked from 5072).